Adding explanatory text to _infographics/emissions/fossil-fuels-emissions.md

willemfrishert opened the pull request:
Task of #14: Add English texts to the published infographics

### Preview Build

Current preview build is available at https://en-facts-on-climate--preview-fossil-fuels-emissions-hyy0cy2f.web.app/infographics/fossil-fuels-emissions

### What does this PR do?

- bulk-translated the Czech text to English using [DeepL](https://www.deepl.com) for fossil-fuels-emissions.md
- Use [Grammarly](https://www.grammarly.com/) to proofread the texts.
- Introduce Glossary for English to support reference to `ipcc` and `phase-out`.
  - Note: this is branched off of eeef83efaf3b8cdc2c2c6410c39c29506f465653 which introduced the glossary for 
- Changed the quote and the link to the Paris Agreement to use https://unfccc.int/files/essential_background/convention/application/pdf/english_paris_agreement.pdf (instead of https://www.mzp.cz/C1257458002F0DC7/cz/parizska_dohoda/$FILE/OEOK-Cesky_preklad_dohody-20160419.pdf).


### Commented out links:
- Commented out preview-box (using Jekyll comments) as these are (not yet) available in English
  -  [slug="elektrina-svet - https://faktaoklimatu.cz/infografiky/elektrina-svet](https://faktaoklimatu.cz/infografiky/elektrina-svet)
  -  [slug="emise-vyroba-cementu - https://faktaoklimatu.cz/explainery/emise-vyroba-cementu](https://faktaoklimatu.cz/explainery/emise-vyroba-cementu)

- Link mentioned in Gas section is not working:
  - "Extraction and transportation: In total, [4000 billion m<sup>3</sup>](https://www.bp.com/content/dam/bp/business-sites/en/global/corporate/pdfs/energy-economics/statistical-review/bp-stats-review-2019-natural-gas.pdf) were extracted in 2019, i.e. about 3.2 billion tons of natural gas. 
  - This also applies to [https://faktaoklimatu.cz/infografiky/emise-fosilni-paliva](https://faktaoklimatu.cz/infografiky/emise-fosilni-paliva) where the link is broken. See https://github.com/faktaoklimatu/web-cz/issues/1130

### Todo:
- [x] Update link bp-stats-review-2019-natural-gas.pdf  once https://github.com/faktaoklimatu/web-cz/issues/1130 is resolved.
  - Updated this together with text corrections.

willemfrishert committed:
Adding ipcc to _data/glossary.yml.

willemfrishert committed:
Adding phase-out to _data/glossary.yml.

willemfrishert committed:
Update _infographics/climate/fossil-fuels-emissions.md (Adding explanatory text). Translated + initial text styling.

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
 ---
 layout:        infographic
-title:         "Global CO2 emissions: fossil fuels & cement production"
+title:         "Global CO₂ emissions: fossil fuels & cement production"
```
Title: I would encourage no colon (to match CZ text and also for flow):

Global CO₂ emissions from fossil fuels & cement production

tjarnikova replied:
Comment on infographic text: contextual sentence missing/imprecise. 

Infographic: 
Emissions of greenhouse gasses are usually converted to their CO2 equivalent (CO2eq) due to the presence of multiple gasses. However, in this case, only CO2 emissions are relevant, and so we include only CO2. 

(Starting a sentence with But is typically inappropriate in a professional text.)

willemfrishert replied:
I updated the title (removed colon).

willemfrishert replied:
As for the text inside the actual Infographic. I don't know if I should be editing the fossil-fuels-emissions.pdf directly. We'd want to make sure we update it at the source so it remains corrected and is not overwritten.

Maybe @mukrop can provide some input if we want/need to change the pdf itself

mukrop replied:
I'd not change the text in the infographic now. There is a separate repository with source files (Adobe Illustrator). although I agree Tereza's suggestion is slightly better, I don't think it's worth doing the change in the PDF with the process around it.

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+## How to read this graph?
+
+* The graph shows **worldwide emissions from burning coal, oil and natural gas** and, for comparison, **emissions from cement production**. Total emissions from these sources for 2019 were 35,897 million tons of CO<sub>2</sub>.
+* The graph only shows emissions from the chemical reaction CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> as emissions from cement production. Emissions created by burning coal or gas for heating are counted as emissions of the respective fuel.
```
→ ⟶ would switch out arrow styles (in CZ version also) for cleanness in equations.

willemfrishert replied:
Change from `⟶` to `→`

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+
+### Coal
+
+* **Usage:** Primarily for the production of electricity and heat ([Mapped worlds coal power plants](https://www.carbonbrief.org/mapped-worlds-coal-power-plants)) in the iron or glass smelting industry or other heating.
```
would change to [Map of world's coal power plants]

willemfrishert replied:
Thanks, I corrected it to `[Map of world's coal power plants]`

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+### Coal
+
+* **Usage:** Primarily for the production of electricity and heat ([Mapped worlds coal power plants](https://www.carbonbrief.org/mapped-worlds-coal-power-plants)) in the iron or glass smelting industry or other heating.
+* **Mining and Transportation:** In 2019, the world mined approximately [8 billion tons of coal](https://www.iea.org/reports/coal-information-overview). China (46% of world production) mined by far the most coal, followed by India (10%), the USA (8%), Indonesia (8%) and Australia (6%). Countries need to import coal to the point of consumption, unlike pipelines which help transport oil and natural ...
```
Misleading translation/ subject-object mismatch. 
Change sentence begining wtih "Countries need to" to "Unlike oil and natural gas, which may be transported by pipelines, coal must be brought to the point of consumption by vehicles. "

willemfrishert replied:
I changed from `"Countries need to..."` to `"Unlike oil and natural gas, which may be transported by pipelines, coal must be brought to the point of consumption by vehicles."`

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+
+### Petroleum
+
+* **Usage:** Most of the oil – or, more accurately, the gasoline and diesel produced from oil – is burned in transportation. It is used for the production of electricity and heat rather exceptionally.
```
Change to:

"The majority of oil – or, more accurately, the gasoline and diesel produced from oil – is burned in transportation. The use of oil for heating is relatively rare."

willemfrishert replied:
Updated as suggestion to "The majority of oil – or, more accurately, the gasoline and diesel produced from oil – is burned in transportation. The use of oil for heating is relatively rare"

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+### Petroleum
+
+* **Usage:** Most of the oil – or, more accurately, the gasoline and diesel produced from oil – is burned in transportation. It is used for the production of electricity and heat rather exceptionally.
+* **Extraction and transportation:** The world produces about [100 million barrels of oil](https://www.eia.gov/outlooks/steo/report/global_oil.php) daily, or about 5 billion tons per year (2019). Most oil is produced in the countries of the Persian Gulf (about 32% of world production), followed by the USA (15%) and Russia (13%). While the US covers its consumption with production, most other majo ...
```
Ships [transport about 61% of crude oil by sea](https://talkbusiness.net/2017/08/61-of-global-crude-oil-and-petroleum-products-transported-by-sea/) and the rest through [ pipelines](http://worldmap.harvard.edu/data/geonode:global_oil_pipelines_7z9).

Approximately 61% of crude oil is transported by ship [link], with the remainder transported by pipelines [link].

This translation isn't the most exact to the Czech original, but I think "transported by ship" is less meaning-ambiguous than "transported by sea". Sentence structure needed modification.

willemfrishert replied:
The sentence structure being modified made sense. I also changed the word `sea` to `ship` as suggested but I was not sure to what extend we want to stay true to the original.

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+%}
+{% endcomment %}
+
+* Heating a mixture of limestone and clay to temperatures of around 1450 °C produces cement. Additionally, the heat causes the limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to  ...
```
Switch sentence subject for legibility: 
Cement is produced by heating a mixture of limestone and clay to temperatures of around 1450 °C. This heating also causes limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat → CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to coal or gas combustion in production. In the graph, the category *cement* only shows emissions from the chemical reaction. The emissions related to coal and gas combustion in cement production are reported under the categories for those fossil fuels.

willemfrishert replied:
Agreed. I also made sure to use the correct arrow here as I updated the text (so we should avoid `⟶` and use `→`)

tjarnikova commented on collections/_infographics/emissions/fossil-fuels-emissions.md:
```diff
@@ -1,16 +1,60 @@
+{% endcomment %}
+
+* Heating a mixture of limestone and clay to temperatures of around 1450 °C produces cement. Additionally, the heat causes the limestone to decompose into calcium oxide and carbon dioxide. Approximately half of the total CO<sub>2</sub> emissions from cement production arise from the CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> chemical reaction. The other half of the total emissions relates to  ...
+* Total emissions from cement production (including heating fuels) are approximately 2.8 billion tons of CO<sub>2</sub>, or about 8% of world CO<sub>2</sub> production. If the cement industry were a state, its emissions would be the third largest in the world, after China and the USA.
```
The total emissions from cement production (including heating fuels) are approximately 2.8 billion tons of CO<sub>2</sub>, or about 8% of world CO<sub>2</sub> production. If the cement industry were a state, its emissions would be the third largest in the world, after China and the USA.

willemfrishert replied:
I updated the text but had a question here with regards to the word `"state"`.
In the sentence: "If the cement industry were a `state`, its ...".

Should we be using `state or this be `country`?

I noticed deepl.com sometimes using the word `state` and other times using `country`. Google translate does the same thing. Note: For a nation or territory considered as an organized political community under one government such as the "United States of America" and the "European Union" the word "state" or "member states" this makes sense.

mukrop replied:
I do agree with Willem that "country" is a better term here. "State" is probably also a residuum of the Czech  language (we use the direct translations of "state" and "country" almost interchangeably).

tjarnikova reviewed (changes requested):
Looks good overall! Most comments address minor but necessary changes (for clarity and legibility).

willemfrishert committed:
Update title

willemfrishert committed:
Update arrow in equation.

willemfrishert committed:
Correct text.

Fix translation mistake 'Mapped worlds coal power plants' to 'Map of world's coal power plants'.

willemfrishert committed:
Correct text.

Misleading translation/subject-object mismatch.

willemfrishert committed:
Correct text.

Correct petroleum section on usage.

willemfrishert committed:
Correct text.

Correct petroleum section on extraction and transportation.

willemfrishert committed:
Update link to external report.

willemfrishert committed:
Correct text.

Switch sentence subject for legibility.

mukrop committed:
Proofreading

mukrop commented:
A small technical comment for @willemfrishert: As we prefer linear history (and rebase PRs instead of merging them), doing an update to current `master` is better done by rebasing the current branch rather than merging from master. If you need more details to understand what I mean, feel free to write me on Slack :-).
